src/RpcClient/Models/RpcVersion.cs

```diff
@@ -44,7 +44,8 @@ public JObject ToJson()
                 json["initialgasdistribution"] = InitialGasDistribution;
                 json["hardforks"] = new JArray(Hardforks.Select(s => new JObject()
                 {
-                    ["name"] = s.Key,
+                    // Strip HF_ prefix.
+                    ["name"] = s.Key.ToString().Substring(3),
```

Comment:
why?

Reply:
See the https://github.com/neo-project/neo-modules/pull/838#issuecomment-1761277835 and updated version.